go-backend/db/user.go

```diff
@@ -2,13 +2,24 @@ package db
 
 import (
 	"context"
+	"time"
 
 	logger "github.com/sirupsen/logrus"
 )
 
 type User struct {
-	Name string `db:"name" json:"full_name"`
-	Age  int    `db:"age" json:"age"`
+	ID              int64     `db:"id" json:"id"`
+	Name            string    `db:"name" json:"full_name"`
+	Age             int       `db:"age" json:"age"`
```

Comment:
We need to remove `Age` here. We don't need to track it in the application and may have legal/hr implications in some areas.

Reply:
Nice catch! it was extra attribute not being used in user schema. Removed in 60298da0af68584d9c932aa3d2b361cbf8cb0f40